pages/vi/team.md

```diff
@@ -27,5 +27,6 @@
 |[![](https://avatars1.githubusercontent.com/u/33208073?s=25)](https://github.com/darthnoward)[darthnoward](profiles/darthnoward.md)|2020-05-02|
 |[![](http://github.com/saura8h.png?size=25)](https://github.com/saura8h)[saura8h](profiles/saura8h.md)|2020-05-02|
 |[![](https://avatars1.githubusercontent.com/u/33208073?s=25)](https://github.com/warthog16)[warthog16](profiles/warthog16.md)|2020-05-17|
+|[![](https://avatars1.githubusercontent.com/u/33208073?s=25)](https://github.com/ali18997)[ali18997](profiles/ali18997.md)|2020-05-23|
```

Comment:
@ali18997 https://avatars1.githubusercontent.com/u/33208073?s=25 is not your github avatar ;)
please fix that

Reply:
@dogi oh sorry about that, fixed it :)